packages/ui/react-ui-editor/src/extensions/comments.ts

```diff
@@ -505,6 +514,22 @@ export const scrollThreadIntoView = (view: EditorView, id: string, center = true
   }
 };
 
+/**
+ * Query the editor state for the active formatting at the selection.
+ */
+export const doesSelectionOverlapComment = (state: EditorState): boolean => {
```

Comment:
selectionContainsComment?

Reply:
separately, perhaps we should disable comments if there's more than one selection range?